drivers/mrf24j40/mrf24j40_internal.c

```diff
@@ -36,6 +36,56 @@ static inline void getbus(mrf24j40_t *dev)
     spi_acquire(SPIDEV, CSPIN, SPI_MODE_0, dev->params.spi_clk);
 }
 
+void mrf24j40_enable_pa_lna(mrf24j40_t *dev)
```

Comment:
If `mrf24j40_enable_lna` is a subset up `mrf24j40_enable_pa_lna` I would have expected it to be called from here.